src/contracts/interfaces/IAVSDirectory.sol

```diff
@@ -17,7 +25,25 @@ interface IAVSDirectory is ISignatureUtils {
     event AVSMetadataURIUpdated(address indexed avs, string metadataURI);
```

Comment:
We should have `updateAVSMetadataURI` and events updated with operatorSets so AVSs can update accordingly to their operatorSets. Especially needed for the "meta" AVSs who need operator sets for UI purposes

Reply:
@stevennevins is leading design for operator set metadata, assuming it will be in separate PR